### PR TITLE
ASHRAE 140 runs improvement

### PR DESCRIPTION
**Pull request by bpark1327**
## Pull Request Description

- Update radiative/convective split for ASHRAE 140 runs
- Increase the number of `misc plug loads` schedule decimal places for ASHRAE 140 runs
- Specify the exact attic volume to match ASHRAE 140 spec.

## Checklist

Not all may apply:

- [ ] EPvalidator.xml has been updated
- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI

### CODE REVIEW

**Comment by bpark1327** [comment]
All ASHARE 140 runs pass the acceptance criteria as of 10/26/2020
[ASHRAE 140 Acceptance Criteria - Heating.xlsx](https://github.com/NREL/OpenStudio-HPXML/files/5427421/ASHRAE.140.Acceptance.Criteria.-.Heating.xlsx)
[ASHRAE 140 Acceptance Criteria - Cooling.xlsx](https://github.com/NREL/OpenStudio-HPXML/files/5427424/ASHRAE.140.Acceptance.Criteria.-.Cooling.xlsx)

File: HPXMLtoOpenStudio/resources/misc_loads.rb [hunk 1]
```diff
@@ -27,7 +27,15 @@ def self.apply_plug(model, plug_load, obj_name, living_space)
     if plug_load.location == HPXML::LocationExterior
       # Set all heat gain as lost
       sens_frac = 0.0
-      lat_fract = 0.0
+      lat_frac = 0.0 # FIXME: is lat_fract intended?
```
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/misc_loads.rb:30]: Good catch, looks like a typo to me. Feel free to remove the FIXMEs here and below.

File: HPXMLtoOpenStudio/resources/misc_loads.rb [hunk 2]
```diff
@@ -27,7 +27,15 @@ def self.apply_plug(model, plug_load, obj_name, living_space)
...
       # Set all heat gain as lost
       sens_frac = 0.0
-      lat_fract = 0.0
+      lat_frac = 0.0
+    end
+
+    if apply_ashrae140_assumptions
+      # ASHRAE 140, Table 7-9: Sensible load accounts for 0.822 of the total load and latent load accounts for 0.178. Sensible loads are 70% radiative and 30% convective.
+      rad_frac = 0.7 * 0.822
+      lat_frac = 0.178
```
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/misc_loads.rb:36]: We are already specifying some of these values in HPXML file, right? So all you need here is `rad_frac = 0.7 * sens_frac` and can leave out `lat_frac` (let it pass through).

**Review by shorowit — APPROVED** [review]
Just need to make this one change and then we can merge this.

**Pull request merged**